bundles/org.openhab.binding.ferroamp/src/main/java/org/openhab/binding/ferroamp/internal/EsoParameters.java

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) 2010-2024 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+package org.openhab.binding.ferroamp.internal;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * The {@link EsoParameters} is responsible for all parameters regarded to ESO
+ *
+ * @author Örjan Backsell - Initial contribution
+ *
+ */
+
+@NonNullByDefault
+public class EsoParameters {
+    public String jsonPostEso;
+
+    public EsoParameters(String jsonPostEso) {
```

Comment:
1. This method does not seem to be used anywhere.
2. Use the defined FerroampBindingConstants where possible.

Reply:
20241107
Deleted this class, it's not used anymore

Reply:
Still there.

Reply:
Unfortunately not.

Reply:
They are gone.